core/app/models/spree/wallet/add_after_order_complete.rb

```diff
@@ -0,0 +1,25 @@
+class Spree::Wallet::AddAfterOrderComplete
```

Comment:
I no this name too strictly named after where it's called from. I think it should be named more like what it does. My English vocabulary is limited, so I only came up with `Spree::Wallet::Storer`. But, besides that I really like the way we are going here. Configurable POROs that are responsible for the actual implementation. 👍